dist/@types/device_interface.d.ts

```diff
@@ -50,6 +50,8 @@ export declare abstract class DeviceInterface {
     abstract removeRawDatabasePayloadWithId(id: string, identifier: ApplicationIdentifier): Promise<void>;
     abstract removeAllRawDatabasePayloads(identifier: ApplicationIdentifier): Promise<void>;
     abstract getNamespacedKeychainValue(identifier: ApplicationIdentifier): Promise<any>;
+    /** Allows SNJS to set the top level keychain value */
+    abstract legacy_setRawKeychainValue(value: any): Promise<void>;
```

Comment:
I'm not sure I understand how to implement this on web. What version does legacy refer to? Should it be named something else, like `v1_0_0_setRawKeychainValue` instead?

Reply:
Yeah maybe it should be prefixed with version. Although I'm not sure if version makes sense because we're accessing this from the latest code. Legacy just means in this case "we don't do this anymore (but your code can if it really has to)". Maybe it should be called `unsafe_setRawKeychainValue` or something like that?

Reply:
I think `legacy` is definitely a better candidate than `unsafe`, as the latter doesn't necessarily indicate that it has a deprecated/obsolete behavior.
The reason I'm suggesting a version number is, what happens if we change `setRawKeychainValue` again in the future?  A form of time-stamping prevents that question from being asked. But maybe we can cross that bridge when we get to it.